feflow/protocols/protein_mutation.py

```diff
@@ -2,14 +2,51 @@
 Implementation of protein mutation protocol based on Nonequilibrium Cycling, using OpenMM as
 MD engine.
 """
+from pathlib import Path
```

Comment:
Are we really going to have a different setupunit / protocol for this? I thought the aim was to just make sure our current Protocols could just handle this use case?

Reply:
Yes probably, for now I create a basic skeleton for it, but we would probably use the same `SetupUnit` for all the protocols as possible. I can remove this placeholder if that makes it better.